.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "semi": false,
-  "singleQuote": true
+  "singleQuote": true,
+  "endOfLine": "auto"
```

Comment:
This is good.May I also suggest adding .editorconfig ? https://editorconfig.org/